src/components/footer/index.tsx

```diff
@@ -1,4 +1,7 @@
 import React from 'react'
+import svgHashicorpLogo from '!!raw-loader!@hashicorp/mktg-logos/corporate/hashicorp/primary/black.svg'
```

Comment:
What does the `!!raw-loader!` prefix on here do? It's not something I did for the logo in navigation-header, so I'm curious if that needs to be updated or not. Haven't seen this yet!

Reply:
Does the same as `?include`, but doesn't require fancy webpack stuff. I think we should be consistent with the former, this is an old habit, so have updated to use `?include` 👍